blockstack_client/snv.py

```diff
@@ -897,32 +971,30 @@ def snv_lookup(verify_name, verify_block_id, trusted_serial_number_or_txid_or_co
         # but that's okay--if the consensus hash in this tx is inauthentic, it will be unreachable
         # from the other consensus hash [short of a SHA256 collision])
         trusted_block_id = get_block_from_consensus(trusted_consensus_hash, proxy=proxy)
-
-
-    elif len(trusted_serial_number_or_txid_or_consensus_hash) == 32 and is_hex(trusted_serial_number_or_txid_or_consensus_hash):
+    elif hash_len_32 and hash_is_hex:
         # consensus hash
         trusted_consensus_hash = trusted_serial_number_or_txid_or_consensus_hash
         trusted_block_id = get_block_from_consensus(trusted_consensus_hash, proxy=proxy)
-        if type(trusted_block_id) == dict and 'error' in trusted_block_id:
+        if isinstance(trusted_block_id, dict) and 'error' in trusted_block_id:
             # got error back
             return trusted_block_id
-
-
-    elif len(trusted_serial_number_or_txid_or_consensus_hash.split("-")) == 2:
+    elif hash_parts_2:
         # must be a serial number
-        parts = trusted_serial_number_or_txid_or_consensus_hash.split("-")
+        parts = trusted_serial_number_or_txid_or_consensus_hash.split('-')
         try:
             trusted_block_id = int(parts[0])
+            # TODO: Check why this variable is unused
             trusted_tx_index = int(parts[1])
         except:
-            log.error("Malformed serial number '%s'" % trusted_serial_number_or_txid_or_consensus_hash)
+            log.error('Malformed serial number "{}"'.format(trusted_serial_number_or_txid_or_consensus_hash))
             return {'error': 'Did not receive a valid serial number'}
 
         trusted_tx = serial_number_to_tx(trusted_serial_number_or_txid_or_consensus_hash, bitcoind_proxy)
```

Comment:
It shouldn't affect correctness, but the idea behind `trusted_block_id` and `trusted_tx_index` was to verify that `trusted_serial_number_or_txid_or_consensus_hash` was a serial number (i.e. $block_height-$tx_index).  We can replace `trusted_serial_number_or_txid_or_consensus_hash` with `'{}-{}' % (trusted_block_id, trusted_tx_index)` in this branch of the conditional.